src/Controllers/SearchController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ModelRepoBrowser.Models;
+
+namespace ModelRepoBrowser.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class SearchController : ControllerBase
+{
+    private readonly ILogger<SearchController> logger;
+
+    private readonly RepoBrowserContext context;
+
+    public SearchController(ILogger<SearchController> logger, RepoBrowserContext context)
+    {
+        this.logger = logger;
+        this.context = context;
+    }
+
+    [HttpGet]
+    public async Task<IEnumerable<Model>> Get(string q)
```

Comment:
REQ: Die Methode zu etwas aussagekräftigerem umbenennen. Vorschlag:
```suggestion
    public async Task<IEnumerable<Model>> Search(string query)
```